src/renderer/components/wallet/assets/AssetsTableCollapsable.tsx

```diff
@@ -145,7 +145,12 @@ export const AssetsTableCollapsable: React.FC<Props> = (props): JSX.Element => {
       const oPrice = getPoolPriceValue(balance, poolDetails, pricePool.poolData)
       const label = FP.pipe(
         oPrice,
-        O.map((price) => formatAssetAmountCurrency({ amount: baseToAsset(price), asset: pricePool.asset, decimal: 3 })),
+        O.map((price) => {
+          if (isEthAsset(balance.asset)) {
+            price.decimal = ETH_DECIMAL
+          }
```

Comment:
Pls keep in mind: If we have to do similar change for `price.decimal` at other places for ETH, extract that logic into a helper for pricing (into `getPoolPriceValue` in this case) - just to use same logic everywhere then ...

Reply:
ok 👍